docs/tutorial/2-tilt-up.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Tilt Up, Up, And Away
```

Comment:
FWIW, one of the user interviewees this week was confused by this title:

> And "Tilt Up, Up, and Away"...Um, I'm not sure what to expect in this section. But...maybe Overview would explain that. But since I don't have Overview to actually see [NOTE: since Prototype doesn't include page text]. But, I have no idea what this "Tilt Up, Up, and Away" would give me. 

Personally, I think it's cute and fairly self-explanatory when you click into it. But it is decidedly the cheekiest of the titles.

Reply:
That's good feedback, and I think totally valid: if you've never used Tilt before you're unlikely to know that `tilt up` is the command and get the (weak) joke.

Thoughts on new title? I'm struggling because while something like "Launching Tilt" sounds the most natural, this section covers a lot about "fundamental concepts" of Tilt as well 🤔

Reply:
Went with "Launching & Managing Resources", we can further rename if needed